features/steps/deciders.py

```diff
@@ -0,0 +1,64 @@
+class StepDecider:
+    """
+    Determines at the step level if a test should be run.
+    """
+
+    def __init__(self, context):
+        """
+        Behave context of currently running step.
+        :param context: 
+        """
+        self.context = context
+
+    def __skip_context(self):
+        return self.context.vendor_skip
+
+    def should_run_test(self):
+        return not self.__skip_context()
+
+
+class ResourceDecider:
+    """
+    Determines if a resource should be validated based on Argonaut Value Sets.
+    """
+    argo_vital_codes = ["8716-3",
```

Comment:
This should potentially use the bloom filter to find the valueset that we need to validate.